packages/llmRunner/src/app/Badge.tsx

```diff
@@ -0,0 +1,18 @@
+import clsx from "clsx";
+import { FC, PropsWithChildren } from "react";
+
+// TODO - move to @quri/ui?
```

Comment:
I'm happy to be a bit slow for this. These items are pretty small, and consistency isn't very critical for our app.